### DISCUSSION TITLE
storybook add message to interaction testing?

### DISCUSSION THREAD

**Discussion by rssilvaba** [storybookjs/storybook#21044] [Help]
I was wondering if it is possible to add a message instead of showing the values when the interactions tests are run in storybook.

for example:

```js
Button.play = async ({ canvasElement }) => {
  const buttonEl = getByRole('button')
  await expect(buttonEl.textContent).toContain('some button text');
  //this gives `expect('some button text').toContain('some button text')`
  //would have been better if it gave: `expect textContent to contain 'some button text'`;
```

any clue if that is possible? FYI it is not a deal breaker for me, it a cheaper alternative to have the test-runner and it works on a case by case, just want to know if there is a way to put extra message when showing the test result.

**Comment by scottrippey** [comment]
This can be done now via `expect(buttonEl).toHaveTextContent('some button text')`.
This will result in a log like `expect(getByRole('button')).toHaveTextContent('some button text')`

**Reply by rssilvaba** [reply]
Oh cool. Thanks for the update. When you say "now" you mean storybook 7 right?

**Reply by scottrippey** [reply]
I know it works in my storybook 7 app, but I think this is a feature of `@storybook/jest` and has been supported for some time.

**Reply by rssilvaba** [reply]
I see what you mean now. The method used here makes all the difference. Instead of reading the textcontent I can call it from the method and that will be reflected in the test. I think I got it now.
